godot-mod-export-tool-plugin/godot-3/project.godot

```diff
@@ -8,6 +8,214 @@
 
 config_version=4
 
+_global_script_classes=[ {
+"base": "ChallengeData",
+"class": "CLChallengeData",
+"language": "GDScript",
+"path": "res://mods-unpacked/Darkly77-ContentLoader/challenges/cl_challenge_data.gd"
```

Comment:
you might want to work in a different project with the addon symlinked into it, otherwise we will get all of these random modded classes

Reply:
true